nengo/utils/connection.py

```diff
@@ -57,3 +57,49 @@ def function(x):
     return {'function': function,
             'eval_points': eval_points,
             'scale_eval_points': False}
+
+
+def eval_point_decoding(conn, sim, eval_points=None):
+    """Get the targets and actual decoded values for a set of eval points.
+
+    This function evaluates the static decoding (i.e. using the neuron type's
+    `rates` function) of a connection for a given set of evaluation points.
+
+    Parameters
+    ----------
+    conn : Connection
+        The Connection to evaluate the decoding of.
+    sim : Simulator
+        A Nengo simulator storing the built connection.
+    eval_points : array_like (N, E) (optional)
+        An N x E array of evaluation points to evaluate the decoding for, where
+        N is the number of points and E is the dimensionality of the input
+        ensemble (i.e. `conn.size_in`). If None (default), use the connection's
+        training evaluation points.
+
+    Returns
+    -------
+    eval_points : ndarray (N, E)
+        A shallow copy of the evaluation points used. E is the dimensionality
+        of the connection input ensemble (i.e. `conn.size_in`).
+    targets : ndarray (N, D)
+        The target function value at each evaluation point.
+    decoded : ndarray (N, D)
+        The decoded function value at each evaluation point.
+    """
+    from nengo.builder.ensemble import get_activities
+    from nengo.builder.connection import get_targets
```

Comment:
Something was breaking when I put these imports at the top of the file, so I put them here.